Taskfile.yml

```diff
@@ -3,21 +3,33 @@
 version: '3'
 
 env:
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2023"
   SRC_DIR: "{{.TASKFILE_DIR}}/src"
 
 tasks:
+  ci:
```

Comment:
Just thinking out loud here, in the case of running CI, do we really want to pull the latest everything? I am asking because it might be better to exclude `task workspace`.

Reply:
Yes, I believe we covered everything in a discussion offline but lmk if I missed something